packages/types/src/documents/app/table.ts

```diff
@@ -49,4 +49,10 @@ export interface Table extends Document {
   sourceId?: string
   relatedFormula?: string[]
   constrained?: string[]
+  _id?: string
+  _rev?: string
```

Comment:
We already get these base fields `_id`, `_rev`, `createdAt` and `_updatedAt` from the base `DocumentType`

Reply:
Solved